src/frontend/typer/skjema.ts

```diff
@@ -73,7 +76,7 @@ export interface IOmBarnetUtvidetFeltTyper {
     borFastMedSøker: ESvar | null;
     skriftligAvtaleOmDeltBosted: ESvar | null;
     sammeForelderSomAnnetBarn: string | null;
-    registrerteUtenlandsperioder: IUtenlandsperiode[];
+    registrerteUtenlandsperioderBarn: IUtenlandsperiode[];
```

Comment:
Småpirk men kanskje denne kunne være prefixes i stedet for postfixet sånn som andre forelder 😊